### PR TITLE
Add docker-compose build

### PR DESCRIPTION
**Pull request by giorgiosironi**
Mostly to setup a basic Travis CI build.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
This will probably be ignored until the `.travis.yml` file is in the `libero:master` branch? /cc @elrossco22

**Comment by elrossco22** [comment]
@giorgiosironi depends on travis settings, we should set it thus though

**Comment by giorgiosironi** [comment]
This will fail at the moment, so it can wait for a `master` branch to have a `docker-compose.yml` configuration (or a `Dockerfile`), then be rebased to be tested.

**Comment by thewilkybarkid** [comment]
Depends on #4.

**Comment by thewilkybarkid** [comment]
Any way not to have 2 checks appearing?

**Comment by giorgiosironi** [comment]
> double status

They are effectively the same build on `travis-ci.com`, one coming from the Github application and one directly as a commit status check from the other side. A general problem, maybe they have some configuration to tweak.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+services: docker
+
+install:
+    - docker-compose build
+
+before_script:
+    - docker-compose up -d
+
+script:
+    - nc -z localhost 8080
```
Comment (giorgiosironi) [inline .travis.yml]: anecdotally checking for listening services should work as soon as the `docker-compose` host port is open, so it's a smoky test but enough for now

**Comment by giorgiosironi** [comment]
Let's :shipit: so that further changes can be tested.

**Pull request merged**